examples/mnist/quantized.cpp

```diff
@@ -1,124 +0,0 @@
-/*
```

Comment:
@beru well it uses quantized layers. Instead of removing maybe the model defined [here](https://github.com/tiny-dnn/tiny-dnn/blob/master/examples/mnist/quantized.cpp#L49-L55) can be embed in the main [`mnist/train.cpp`](https://github.com/tiny-dnn/tiny-dnn/blob/master/examples/mnist/train.cpp) file.